components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -44,15 +63,32 @@ subprocess.check_call([
 
 // Configure the Python environments
 envs {
-    bootstrapDirectory = new File(buildDir, 'bootstrap')
-    envsDirectory = new File(buildDir, 'envs')
+    bootstrapDirectory = GLEAN_BOOTSTRAP_DIR
     pipInstallOptions = "--trusted-host pypi.python.org --no-cache-dir"
 
     // Setup a miniconda environment. conda is used because it works
     // non-interactively on Windows, unlike the other options
-    conda "Miniconda3", "Miniconda3-4.5.11", "64", ["glean_parser==${GLEAN_PARSER_VERSION}"]
+    conda "Miniconda3", "Miniconda3-${MINICONDA_VERSION}", "64", ["glean_parser==${GLEAN_PARSER_VERSION}"]
+}
+
+// Setup the python environment before the build starts.
+task createBuildDir {
+    description = "Make sure the build dir exists before creating the Python Environments"
+    onlyIf {
+        !file(buildDir).exists()
+    }
+    doLast {
+        println "Creating build directory:" + buildDir.getPath()
+        buildDir.mkdir()
+    }
 }
 
+// Make sure that a build directory exists before attempting to download
```

Comment:
Is this comment still valid? At the very least, we should mention that this is failing because we're not creating the dir under the project build dir, right?

Reply:
Sure.  I'll clarify that we're talking about the project's build directory here.